### PR TITLE
update download.rst to reference sourceforge, also restructured text, would appreciate a review.

### PR DESCRIPTION
**Pull request by camerons**

### CODE REVIEW

File: en/download.rst [hunk 1]
```diff
@@ -6,33 +6,31 @@
 Download
 ================================================================================
 
-The latest stable release of |osgeo-live-version| :
+The latest stable release is |osgeo-live-version| and can be downloaded from:
+https://sourceforge.net/projects/osgeo-live/files/
 
-.. image:: ../images/download_buttons/download-dvd.png
```
Comment (kalxas) [inline en/download.rst]: Do we really want to remove the images from here?

File: en/download.rst [hunk 2]
```diff
@@ -6,33 +6,31 @@
...
+**osgeo-live iso**: |osgeolive-iso-size|
 
-.. image:: ../images/download_buttons/download-dvd.png
-  :alt: Download 64 bit iso file
-  :align: left
-  :target: http://download.osgeo.org/livedvd/release/9.5/osgeo-live-9.5-amd64.iso/download
+A Lubuntu based bootable image, which can be copied to an 8 Gig USB thumb drive (faster and recommended) or DVD (cheaper). This image includes windows and mac installers for many of the applications, and is the image you would typically use if handing out OSGeo-Live to new users at a conference.
 
-`osgeo-live-9.5-amd64.iso <http://download.osgeo.org/livedvd/release/9.5/osgeo-live-9.5-amd64.iso/download>`_ : |osgeolive-iso-size|
+**osgeo-live-mini iso**: |osgeolive-iso-mini-size|
```
Comment (kalxas) [inline en/download.rst]: There is no mini iso anymore, is has been renamed to the main iso and it does not include the mac and win installers. This is what we decided some months ago.

**Comment by kalxas** [comment]
Also, this pull request includes the code from the previous pull request, changing the makefile too. Is this correct?

File: en/download.rst (hunk 1 above)
Comment (camerons) [inline en/download.rst]: The current images includes text which doesn't align with the images we offer. If we want to keep images (which I'm ok with), someone will need to update images.

**Comment by camerons** [comment]
Yes, changes to the makefile is not related to this download.rst change. (My immature git skills at play here). However, the makefile changes are safe to apply, they are not effecting normal build functions, just a new "make small" option which quickly makes english docs for quick review.

File: en/download.rst (hunk 2 above)
Comment (camerons) [inline en/download.rst]: Extra commit added to remove reference to mini image.

File: en/download.rst (hunk 1 above)
Comment (kalxas) [inline en/download.rst]: agreed

File: en/download.rst (hunk 2 above)
Comment (kalxas) [inline en/download.rst]: Thanks

**Comment by kalxas** [comment]
No problem, I will apply them in the correct order.

**Pull request merged**